### DISCUSSION TITLE
User logged out after a while

### DISCUSSION THREAD

**Discussion by GrouzyW** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#812] [Q&A]
Hello
On my Macbook Pro M1, if I leave my computer on charge for a while (overnight for instance), my user which is the one running AlDente (admin user) gets logged out completely.
It doesn't look like the Mac as restarted because other logged in account are still logged in.
As a result, the charge just carries on to 100%, although I limited to 80%.

I'm currently using AlDente 1.19.1 with macOS Ventura (but had the same issue with Monterey)

I activated the following options:
Stop charging when sleeping
Automatic discharge
Disable sleep until charge limit
Lock screen when closed

Is that something wrong with the setup, or could it be an issue with AlDente ?

thanks

**Comment by GrouzyW** [comment]
I know this is very old, but just to let everyone knows, I finally found what the culprit was, and it's not AlDente !
It turns out that I had a system setting activated which would logout users after 60 minutes of inactivity.
(System Settings->Privacy & Security->Advanced button at the very bottom)
I have no idea why that was activated to start with, or if it's the default...

**Reply by MatthiasKerbl** [reply]
Hi @GrouzyW,

I am glad you could resolve your issue and thanks for reporting back and letting us know that your issue was not caused by AlDente.

Cheers,
Matthias
